Maple2.Database/Model/Map/UgcMapCube.cs

```diff
@@ -16,13 +17,19 @@ internal class UgcMapCube {
     public float Rotation { get; set; }
 
     public int ItemId { get; set; }
+    public HousingCategory HousingCategory { get; set; }
+    public CubePortalSettings? PortalSettings { get; set; }
```

Comment:
I am wondering if this should be expanded for various cube that has its own settings, such as megaphones or nuturing npcs. Expanded in the same sense as ItemAppearance in Item